src/helpers/gpt.ts

```diff
@@ -54,6 +86,66 @@ Example:[
 ]
 `;
 
+export const getPRSpec = async (context: Context, chatHistory: CreateChatCompletionRequestMessage[], streamlined: StreamlinedComment[]) => {
```

Comment:
```suggestion
export async function getPRSpec(context: Context, chatHistory: CreateChatCompletionRequestMessage[], streamlined: StreamlinedComment[]) {
```

Use named functions, because it is more expressive for debugging in the context of stack traces.